From e53e4cb1ac8f3f59c330f2f9b1241dc6b0ac7357 Mon Sep 17 00:00:00 2001
From: Frode Isaksen <fisaksex@intel.com>
Date: Fri, 4 Jan 2013 15:41:44 +0100
Subject: [PORT FROM MAIN][BT] Enable bluetooth Low Power Mode

BZ: 82570

Re-enable Low Power Mode (LPM), that was disabled
for bluedroid bring-up.

Change-Id: Iaf5c5dcec54d2c804799c3bf6cb0475a7856d2f6
Orig-Change-Id: I6823a7fbf27101dc17f3049b2cd2e2a13b2cfc21
Feature: BT
Category: Integration
Purpose: PV-Enabler
Origin: Internal
Upstream-candidate: no
Signed-off-by: Frode Isaksen <fisaksex@intel.com>
Reviewed-on: http://android.intel.com:8080/85004
Reviewed-by: cactus <cactus@intel.com>
Reviewed-by: Isaksen, FrodeX <frodex.isaksen@intel.com>
Reviewed-by: Aquilone, RaffaeleX <raffaelex.aquilone@intel.com>
Reviewed-by: Bondier, CedricX <cedricx.bondier@intel.com>
Reviewed-by: Gros, Julien <julien.gros@intel.com>
Reviewed-by: Yang, Fei <fei.yang@intel.com>
Tested-by: Zurmely, PierreX <pierrex.zurmely@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
---
 btif/src/btif_core.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/btif/src/btif_core.c b/btif/src/btif_core.c
index 8602d3d..ca39cc8 100644
--- a/btif/src/btif_core.c
+++ b/btif/src/btif_core.c
@@ -580,7 +580,7 @@ void btif_enable_bluetooth_evt(tBTA_STATUS status, BD_ADDR local_bd)
 
     bte_main_postload_cfg();
 #if (defined(HCILP_INCLUDED) && HCILP_INCLUDED == TRUE)
-    bte_main_enable_lpm(FALSE);
+    bte_main_enable_lpm(TRUE);
 #endif
     /* add passing up bd address as well ? */
 
-- 
1.7.4.1

